### ISSUE TITLE
Accessibility : Narrator is reading the 1st option, in the Tag Picker multiple times.

### ISSUE THREAD

**Issue by v-sagsah** [microsoft/fluentui#13290]
**Issue:**
Narrator is reading the 1st option, in the Tag Picker multiple times.

Issue can be reproducible in the below code pen links :
https://codepen.io/sagsah/pen/RwWxxpw?editors=1010
https://codepen.io/slimemold/pen/XWmOVpZ?editors=0010

**Browser Details: ​**
OS: Windows 10 (Version 1903, Build: 18362.175)
**Chromium Edge** Version 81.0.416.77 (Official build) (64-bit)

Please find the steps to repro the issue in the Tag Picker control:
1. Open Edge browser and switch on the Narrator.
2. Open link https://codepen.io/sagsah/full/RwWxxpw
3. Tab till "Filter items in suggestions" textfield and type "b".
4. Now, using down arrow, go to first option "black - multiple - colors - selected - here" as shown below. Do not press Enter.

![Screenshot (112)](https://user-images.githubusercontent.com/64409491/82721234-f83f7a00-9cd8-11ea-99bb-4384005119f6.png)
5. Listen what Narrator is reading.

**Actual:**
Narrator is reading the first option "black - multiple - colors - selected - here"  multiple times.

**Expected:**
It should read the first option only once, like reading the option 2 & 3.

Please find the steps to repro the issue in the People Picker control:
1. Open Edge browser and switch on the Narrator.
2. Open link https://codepen.io/slimemold/full/XWmOVpZ
3. Tab till text-field. The suggestion lists will get populated.
4. Now, using down arrow, go to first option "Annie Lindqvist" as shown below. Do not press Enter.

![Screenshot (113)](https://user-images.githubusercontent.com/64409491/82721337-ea3e2900-9cd9-11ea-9a4a-9bc87864046a.png)
5. Listen what Narrator is reading.

**Actual:**
Narrator is reading the first option "Annie Lindqvist"  multiple times.

**Expected:**
It should read the first option only once, like reading the option 2 & 3.

**Comment by aneeshack4** [comment]
@v-sagsah This is a dupe of #12999 - please read the response on there before filing another.

**Comment by ecraig12345** [comment]
@v-sagsah Do you mean it's reading the first tag you select two times, or something else? It sounds like this *may* be a separate issue from #12999 but it's hard to tell without exact repro steps. This is why we ask you to fill out our whole issue template instead of deleting most of it.

**Comment by v-sagsah** [comment]
@aneeshack4 / @ecraig12345 This issue not a dupe #12999. In this case, the issue was screen reader was reading the place holder value multiple times. But in here, I have not provided any placeholders.

**Browser Details: ​**
OS: Windows 10 (Version 1903, Build: 18362.175)
Chromium Edge Version 81.0.416.68 (Official build) (64-bit)
Google Chrome Version 80.0.3987.149

Please find the steps to repro the issue in the Tag Picker control:
1. Open Edge browser and switch on the Narrator.
2. Open link https://codepen.io/sagsah/full/RwWxxpw
3. Tab till "Filter items in suggestions" textfield and type "b".
4. Now, using down arrow, go to first option "black - multiple - colors - selected - here" as shown below. Do not press Enter.

![Screenshot (112)](https://user-images.githubusercontent.com/64409491/82721234-f83f7a00-9cd8-11ea-99bb-4384005119f6.png)
5. Listen what Narrator is reading.

**Actual:**
Narrator is reading the first option "black - multiple - colors - selected - here"  multiple times.

**Expected:**
It should read the first option only once, like reading the option 2 & 3.


Please find the steps to repro the issue in the People Picker control:
1. Open Edge browser and switch on the Narrator.
2. Open link https://codepen.io/slimemold/full/XWmOVpZ
3. Tab till text-field. The suggestion lists will get populated.
4. Now, using down arrow, go to first option "Annie Lindqvist" as shown below. Do not press Enter.

![Screenshot (113)](https://user-images.githubusercontent.com/64409491/82721337-ea3e2900-9cd9-11ea-9a4a-9bc87864046a.png)
5. Listen what Narrator is reading.

**Actual:**
Narrator is reading the first option "Annie Lindqvist"  multiple times.

**Expected:**
It should read the first option only once, like reading the option 2 & 3.

Hope, I am able to explain you the scenario now. Since i don't have narrator buddy, so couldn't able to show you the speech text. 
It will be helpful, if you could let me know, how can I enable the Narrator buddy tool, in windows 10.

**Comment by v-sagsah** [comment]
@aneeshack4 / @ecraig12345 could you please provide and update on this ?

**Comment by v-sagsah** [comment]
@ecraig12345 could you please provide and update on this ?

**Comment by aneeshack4** [comment]
@v-sagsah I see this reproing 
![image](https://user-images.githubusercontent.com/4161791/83668765-18ebb780-a596-11ea-8d8c-46a60e676460.png)
The person does read get out twice... I actually think it's reading the rest of the people out twice too. @joschect Any idea what could be causing this and what the fix could be?
